code/modules/mob/living/say.dm

```diff
@@ -164,9 +164,11 @@
 		speaking.broadcast(src,trim(message))
 		return 1
 
-	if((is_muzzled()) && !(speaking && (speaking.flags & LANG_FLAG_SIGNLANG)))
-		to_chat(src, "<span class='danger'>You're muzzled and cannot speak!</span>")
-		return
+	if(!speaking || !(speaking.flags & LANG_FLAG_SIGNLANG))
+		var/obj/item/muzzled = is_muzzled()
```

Comment:
I don't know if I'm okay with an `is_\w+` proc returning a datum rather than `FALSE`/`TRUE`. I'd probably use something like `get_item_blocking_mouth()` or something, but it's a bit wordy. Up to you though.